src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -0,0 +1,221 @@
+    frontConfigs =
+        new TalonFXConfiguration()
+            .withSlot0(climberConfigsToClimber)
+            .withSlot1(climberConfigsAtClimber)
+            .withMotorOutput(
+                climberOutputConfigs.withInverted(InvertedValue.CounterClockwise_Positive));
+    backConfigs =
+        new TalonFXConfiguration()
+            .withSlot0(climberConfigsToClimber)
+            .withSlot1(climberConfigsAtClimber)
+            .withMotorOutput(climberOutputConfigs.withInverted(InvertedValue.Clockwise_Positive));
+    climberMotorFront.getConfigurator().apply(frontConfigs);
+    climberMotorBack.getConfigurator().apply(backConfigs);
+    // For simulation
+    climberTargetAngle = ClimberConstants.kClimberRetractedSetpoint;
+    if (Robot.isSimulation()) {
+      simClimberGearbox = DCMotor.getKrakenX60(1);
+      climberSimNormal =
+          new SingleJointedArmSim(
+              simClimberGearbox,
+              ClimberConstants.KGearRatio,
+              ClimberConstants.kClimberArmMOI.in(KilogramSquareMeters),
+              ClimberConstants.kClimberArmLength.in(Meters),
+              ClimberConstants.kClimberArmMinAngle.in(Radians),
+              ClimberConstants.kClimberArmMaxAngle.in(Radians),
+              true,
+              climberTargetAngle.in(Radians));
+      climberSimLifting =
+          new SingleJointedArmSim(
+              simClimberGearbox,
+              ClimberConstants.KGearRatio,
+              SingleJointedArmSim.estimateMOI(
+                  ClimberConstants.kClimberArmLength.in(Meters),
+                  ClimberConstants.kClimberMass.in(Kilograms) / 2),
+              ClimberConstants.kClimberArmLength.in(Meters),
+              ClimberConstants.kClimberArmMinAngle.in(Radians),
+              ClimberConstants.kClimberArmMaxAngle.in(Radians),
+              true,
+              ClimberConstants.kClimberRetractedSetpoint.in(Radians));
+
+      climbMech = new Mechanism2d(2, 2);
+      climbMechRoot = climbMech.getRoot("Climb Mech root", 1, 1);
+      climbMechLigament =
+          climbMechRoot.append(
+              new MechanismLigament2d(
+                  "Climb Mech Ligament",
+                  1,
+                  Radians.of(climberSimNormal.getAngleRads()).in(Degrees)));
+      climbMechTargetLigament =
+          climbMechRoot.append(
+              new MechanismLigament2d(
+                  "Climb Mech Target Ligament",
+                  0.8,
+                  climberTargetAngle.in(Degrees),
+                  10,
+                  new Color8Bit(255, 255, 0)));
+      SmartDashboard.putData("Climb Mech", climbMech);
+    }
+  }
+
+  private Command runClimber(Angle position, int slot) {
+    return runOnce(
+        () -> {
+          climberTargetAngle = position;
+          climberMotorFront.setControl(
+              new PositionDutyCycle(position.times(ClimberConstants.KGearRatio)).withSlot(slot));
```

Comment:
Since your feedforward numbers are in Volts, you should use PositionVoltage instead of PositionDutyCycle